src/drivers/imx/sai.c

```diff
@@ -413,6 +413,12 @@ static int sai_get_hw_params(struct dai *dai,
 	return 0;
 }
 
```

Comment:
Please add more context for this. How is this supposed to work.

Reply:
@dbaluta hw_params is supposed to work in the same fashion like in Linux kernel ALSA layer - the SAI driver is able to get via this callback all the information required to configure the SAI BCLK when in clock provider mode. I updated the pull request, please check. Thank you.